src/components/plots/ContinuousEmbeddingPlot.jsx

```diff
@@ -26,10 +25,9 @@ const ContinuousEmbeddingPlot = (props) => {
     loading, error,
   } = useSelector((state) => state.embeddings[embeddingSettings.method]) || {};
   const geneExpression = useSelector((state) => state.genes.expression);
-  const geneProperties = useSelector((state) => state.genes.propreties);
   const cellSets = useSelector((state) => state.cellSets);
   const [plotSpec, setPlotSpec] = useState({});
-  const PROPERTIES = ['dispersions'];
```

Comment:
there is lots of hardcoding of "dispersions" in many places in the code. This means that in the future, if we wanted to use something else, rather than dispersions, we would have to change all those hardocodings. It will be better if you extract `dispersions` as a const variable (as it was before, but make it a string, not an array) and use that one in the code.